torch/csrc/distributed/c10d/init.cpp

```diff
@@ -3066,6 +3076,28 @@ options :class:`~torch.distributed.ProcessGroupNCCL.Options`).
           .def_readonly("backend", &::c10d::Backend::Options::backend)
           .def_readwrite("_timeout", &::c10d::Backend::Options::timeout);
 
+  intrusive_ptr_class_<::c10d::ProcessGroup::MergeOptions>(
+      module, "MergeOptions", R"(Class for ProcessGroup Merge Options)")
```

Comment:
Thoughts on taking these directly as arguments to merge_remote_group instead of having a separate object? 

I think it's generally nicer for the Python API to avoid the options object though as a hybrid we can expose kwargs at the Python layer and options in C++? Same way we handle the other PG op options